CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ### 3.0.3
 - Optimize association eager loadable checks by caching information on the association's reflection.
+- Optimize association eager loading if we're only eager loading associations for a single model.
```

Comment:
I assume you havent released the gem, otherwise this should go into 3.0.4.

Reply:
Yup. I haven't released 3.0.3 yet.